boards/arm/nrf54l15pdk_nrf54l15/board.cmake

```diff
@@ -1,4 +1,10 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+if(CONFIG_BOARD_NRF54L15PDK_NRF54L15_CPUAPP)
```

Comment:
Is this `if` actually needed? There's no other targets for this board anyway.

Reply:
fixed